tmk_core/common/keyboard.c

```diff
@@ -108,7 +108,7 @@ void matrix_scan_perf_task(void) {
     uint32_t timer_now = timer_read32();
     if (TIMER_DIFF_32(timer_now, matrix_timer) > 1000) {
 #    if defined(CONSOLE_ENABLE)
-        dprintf("matrix scan frequency: %d\n", matrix_scan_count);
+        dprintf("matrix scan frequency: %ld\n", matrix_scan_count);
```

Comment:
```suggestion
        dprintf("matrix scan frequency: %l\n", matrix_scan_count);
```

Reply:
Nope, that doesn't work.  That generates the same error.

Reply:
According to this it should work :/
https://www.tutorialspoint.com/format-specifiers-in-c

Reply:
Well, tested with both avr and arm, and this is what works.  Yay for inconsistent documentation. :D

Reply:
I've updated the suggestion to `%lu` since it's a uint32.